This patch deals with displaying floating ip changes in horizon
--- a/openstack_dashboard/api/neutron.py
+++ b/openstack_dashboard/api/neutron.py
@@ -420,7 +420,6 @@
         ports = port_list(self.request, tenant_id=tenant_id)
         servers, has_more = nova.server_list(self.request)
         server_dict = SortedDict([(s.id, s.name) for s in servers])
-        reachable_subnets = self._get_reachable_subnets(ports)
 
         targets = []
         for p in ports:
@@ -430,8 +429,6 @@
             port_id = p.id
             server_name = server_dict.get(p.device_id)
             for ip in p.fixed_ips:
-                if ip['subnet_id'] not in reachable_subnets:
-                    continue
                 target = {'name': '%s: %s' % (server_name, ip['ip_address']),
                           'id': '%s_%s' % (port_id, ip['ip_address']),
                           'instance_id': p.device_id}
